src/v/config/configuration.h

```diff
@@ -484,4 +490,63 @@ struct convert<model::broker_endpoint> {
     }
 };
 
+template<>
+struct convert<model::cleanup_policy_bitflags> {
+    using type = model::cleanup_policy_bitflags;
+    static Node encode(const type& rhs) {
+        Node node;
+
+        auto compaction = (rhs & model::cleanup_policy_bitflags::compaction)
+                          == model::cleanup_policy_bitflags::compaction;
+        auto deletion = (rhs & model::cleanup_policy_bitflags::deletion)
+                        == model::cleanup_policy_bitflags::deletion;
+
+        if (compaction && deletion) {
+            node = "compact,delete";
```

Comment:
you need `node` normalization like removing all white spaces

Reply:
will fix

Reply:
this is actually encoding into `yaml` I will add normalization to decoding side